extensions/wikia/EditPreview/js/editpage.event.helper.js

```diff
@@ -90,10 +90,37 @@ define('editpage.event.helper', ['wikia.window'], function(window, ace){
 		return $('#categories');
 	}
 
+	function iFrameForm(url) {
+		var object = this;
+		object.time = new Date().getTime();
+		object.form = $('<form action="' + url + '" target="iframe' + object.time + '" style="display:none" method="post" id="form' + object.time + '" name="form' + object.time + '"></form>');
```

Comment:
Please consider using more unique `id`, `form` seams to be to general and could already exist in DOM